joomla/joomla-cms
# [5.1] Update deleted files list in script.php for 5.1.0-beta1 (2)

Pull Request for Issue https://github.com/joomla/joomla-cms/pull/42945#issuecomment-1975174451 .

### Summary of Changes

This pull request (PR) updates the lists of files and folders to be deleted on update in file `administrator/components/com_admin/script.php` by the changes from PR #42945 in the 5.1-dev branch in preparation for the upcoming 5.1.0-beta1 release.

All files and folders are caused by a restructuring of the composer dependency "web-token" coming with the update of that dependency. 

### Testing Instructions

Code review.

Or if you want to make a real test:

On the current 5.1-dev branch as it is right now, build new packages using the following command:
`php ./build/build.php --remote=HEAD --exclude-gzip --exclude-zstd`.

Save the created full installation package somewhere.

Download the full installation package of the latest 5.1 nightly build from tonight.

Compare the 2 packages.

Check the changes made by this PR.

### Actual result BEFORE applying this Pull Request

The files and folders below the "libraries/vendor/web-token" which were removed when comparing the current build with the latest nightly are not in the lists of files and folders in script.php.

### Expected result AFTER applying this Pull Request

The files and folders below the "libraries/vendor/web-token" which were removed when comparing the current build with the latest nightly are in the lists of files and folders in script.php.

### Link to documentations
Please select:
